modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -2636,6 +2860,133 @@ else if (selectExpression instanceof ScalarDSLQueryAlias) {
 		return values;
 	}
 
+	private void _insertIntoLocalizationTable(
+			ObjectDefinition objectDefinition, long objectEntryId,
+			Map<String, Serializable> values)
+		throws PortalException {
+
+		DynamicObjectDefinitionLocalizationTable
+			dynamicObjectDefinitionLocalizationTable =
+				_dynamicObjectDefinitionLocalizationTableFactory.create(
+					objectDefinition);
+
+		String insertIntoLocalizationTableStatement =
+			_createInsertIntoLocalizationTableStatement(
+				dynamicObjectDefinitionLocalizationTable, values);
+
+		if (insertIntoLocalizationTableStatement == null) {
+			return;
+		}
+
+		Map<String, Map<String, String>> i18nObjectFieldValues =
+			new HashMap<>();
+		Set<Locale> locales = new HashSet<>();
+
+		for (Map.Entry<String, Serializable> entry : values.entrySet()) {
```

Comment:
@gabrielwas the fields retrieve here (dynamicObjectDefinitionLocalizationTable.getObjectFields()) always will be localized.

because [here](https://github.com/liferay/liferay-portal/blob/a7f8e3e8bd94187ac6d5a1de5becb55b5c472989/modules/apps/object/object-service/src/main/java/com/liferay/object/internal/petra/sql/dsl/DynamicObjectDefinitionLocalizationTableFactory.java#L42-L43) we're finding by localized true

Reply:
Hi @gabrielwas,

done [here](https://github.com/liferay-objects/liferay-portal/pull/1913/commits/709db3c17e234668161f2eae26bfaf33d5ae3dbe) and [here](https://github.com/liferay-objects/liferay-portal/pull/1913/commits/ae3d61dd1ac05473be933e22ac37fea552878f82)

Reply:
done [here](https://github.com/liferay-objects/liferay-portal/pull/1913/commits/bb3a5b3403af7b7badcbee426a0327d4cde1afb5)

Reply:
and [here](https://github.com/liferay-objects/liferay-portal/pull/1913/commits/dfc52c5f3b3bc11c1bd316c083775094d941c666)